### DISCUSSION TITLE
Difference between waketime and wakapi feature set

### DISCUSSION THREAD

**Discussion by mainrs** [muety/wakapi#305] [Q&A] [answered]
The readme mentions that there is “Partially compatible with WakaTime”. What exactly are the differences?

__Hijacking because I do not want to open another issue/question:__
Is the `heartbeats(.bulk)` API endpoint the only one that wakatime-plugins actually use?

**Comment by muety** [comment, marked as answer]
With regard to features, I once outlined the differences in [this article](https://muetsch.io/time-tracking-for-developers-%E2%80%93-wakatime-vs-hakatime-vs-wakapi.html). 

With regard to API endpoints, check our [Swagger docs](https://wakapi.dev/swagger-ui/) in comparison with WakaTime's API docs.

The only endpoints used by the CLI are the heartbeats and statusbar endpoints, I think.

**Reply by mainrs** [reply]
The article link links back to this discussion :)

**Reply by muety** [reply]
Sorry, fixed the link.
